flang/lib/Optimizer/Builder/FIRBuilder.cpp

```diff
@@ -24,12 +24,7 @@
 #include "llvm/Support/ErrorHandling.h"
 #include "llvm/Support/MD5.h"
 
-static llvm::cl::opt<std::size_t>
-    nameLengthHashSize("length-to-hash-string-literal",
-                       llvm::cl::desc("string literals that exceed this length"
-                                      " will use a hash value as their symbol "
-                                      "name"),
-                       llvm::cl::init(32));
+static constexpr std::size_t nameLengthHashSize = 32;
```

Comment:
This should really be a command-line option. Somehow. (But not in a file in Optimizer.)

Reply:
We might be able to introduce this options with a `PassPipelineOptions`.
https://mlir.llvm.org/docs/PassManagement/#instance-specific-pass-options

Reply:
Can you leave a comment here that this is a TODO to be done one way or another. Thanks.